dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -202,7 +202,7 @@ func addAdditionalFiles(srcPath, dstPath string, hadoopVersion version, version
 			"bin/alluxio-stop.sh",
 			"bin/alluxio-workers.sh",
 			"bin/launch-process",
-			fmt.Sprintf("client/alluxio-%v-client.jar", version),
+			fmt.Sprintf("client/build/alluxio-%v-client.jar", version),
```

Comment:
let's append a hadoop version to the client jar to indicate which hadoop version it was compiled with? i.e. `alluxio-{version}-client-{hadoopVersion}.jar`?

Reply:
hmm i think for the jar itself i can rename it but i cannot rename the artifact/module due to backcompat (it gets published in maven). the symlink name would also need to remain as before